src/Uno.Toolkit.RuntimeTests/Helpers/UnitTestUIContentHelperEx.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if PR_409 // todo@xy: remove conditional when rte#59 && #409 are merged
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UI.RuntimeTests;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+#else
+using Windows.UI.Xaml;
+#endif
+
+using Base = Uno.UI.RuntimeTests.UnitTestsUIContentHelper;
+
+namespace Uno.Toolkit.RuntimeTests.Helpers
+{
+	internal static class UnitTestUIContentHelperEx
```

Comment:
Can we push this extension into the runtimetests repo so that it becomes available for all consumers of the runtimetests

Reply:
Yeah that makes sense :)